apps/arduino_module/module_test/module_test.ino

```diff
@@ -0,0 +1,50 @@
+#include <Arduino.h>
+
+#include <Wire.h>
+#include <stdint.h>
+#include "signpost_api.h"
+
+#define ARDUINO_MODULE_I2C_ADDRESS 0x48
```

Comment:
The end goal is to allow the controller to dynamically assign app ids and i2c addresses during initialization. So eventually we won't have to maintain this.